build/source/engine/derivforce.f90

```diff
@@ -234,11 +236,21 @@ subroutine derivforce(time_data,forc_data,attr_data,mpar_data,prog_data,diag_dat
  dataStep = data_step/secprhour  ! time step (hours)
  ahour    = real(jh,kind(dp)) + real(jmin,kind(dp))/minprhour - data_step/secprhour  ! decimal hour (start of the step)
 
+ ! check slope/aspect intent for radiation calculation
+ if(aspect == -1)then
```

Comment:
Could this be changed to a missing value indicator from `engine/nrtype.f90` to be clearer?

Reply:
done